plugins.d/alarm-notify.sh

```diff
@@ -142,6 +142,12 @@ docurl() {
     return $?
 }
 
+# -----------------------------------------------------------------------------
+# List of all the notification mechanisms we support.
+# Used in a couple of places to write more compact code.
+
+method_names="email pushover pushbullet telegram slack alerta flock discord hipchart twilio messagebird pd fleep syslog custom msteam"
```

Comment:
I suggest to turn this to multi-line with a backslash at the end of each line, so that we can easily track what is added, removed, changed.

Reply:
Agreed, except we don't need backslashes.  Bash supports multi-line strings without backslashes, and a newline will work just fine as a separator for separating the items.